backend/src/main/java/gov/cdc/usds/simplereport/service/idverification/LiveExperianService.java

```diff
@@ -91,16 +94,40 @@ private String fetchToken() {
     requestBody.put("client_secret", _experianProperties.getClientSecret());
 
     HttpEntity<ObjectNode> entity = new HttpEntity<>(requestBody, headers);
-    try {
-      ObjectNode responseBody =
-          _restTemplate.postForObject(
-              _experianProperties.getTokenEndpoint(), entity, ObjectNode.class);
-      if (responseBody == null) {
-        throw new ExperianAuthException("The Experian token request returned a null response.");
+    int retryOn500AuthCounter = 0;
+    int MAX_REFETCH_TRIES = 2;
```

Comment:
a very minor nit: MAX_REFETCH_TRIES should either be made into a constant field or camelCased.

Citation: https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names